drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/test/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorPresenterTest.java

```diff
@@ -284,6 +297,21 @@ public void testMakeMenuBarWithoutUpdateProjectPermission() {
         verify(fileMenuBuilder).addNewTopLevelMenu(alertsButtonMenuItem);
     }
 
+    @Test
+    public void testSave() {
+        presenter.onStartup(XLSPath,
+                            mock(PlaceRequest.class));
+
+        doReturn(Optional.of(mock(WorkspaceProject.class))).when(workbenchContext).getActiveWorkspaceProject();
+        doReturn(promises.resolve(false)).when(projectController).canUpdateProject(any());
+
+        presenter.save("saving");
+
+        verify(metadataService).saveMetadata(any(Path.class),
+                                             any(Metadata.class),
```

Comment:
Could we please replace these `any()` with `eq()` as we do in https://github.com/Rikkola/kie-wb-common/pull/7 ?